bugbug/bug_features.py

```diff
@@ -569,6 +569,30 @@ def __call__(self, bug, **kwargs):
         return get_time_to_assign(bug)
 
 
+def get_time_to_close(bug):
+    """Calculate the time until closure or the time since closure for a bug."""
+    if bug["cf_last_resolved"]:
+        return (
+            parser.parse(bug["cf_last_resolved"]) - parser.parse(bug["creation_time"])
+        ).total_seconds() / 86400
+
+    if len(bug["comments"]) > 0:
+        last_comment_time = bug["comments"][-1]["creation_time"]
+
+        return (
+            datetime.now(timezone.utc) - parser.parse(last_comment_time)
+        ).total_seconds() / 86400
```

Comment:
Why are we doing this?

Reply:
This was meant to retrieve the timestamp of the last comment if the bug wasn't closed. However, it wouldn't be useful anymore, as open bugs are ignored. I'll remove it. 🤓